CONTRIBUTING.md

```diff
@@ -0,0 +1,256 @@
+# Contributing Guide
+
+## Welcome to Percona Server for MySQL!
+
+We're excited to have you join the Percona community and participate in keeping open-source open.
+  
+You can contribute in one of the following ways:
+
+* Submit a bug report or a feature
+   request in [ our public Jira ](https://jira.percona.com/projects/PS/issues).
+* Submit a pull request (PR) with the code patch.
+* Contribute to [the documentation](https://github.com/percona/psmysql-docs/blob/innovation-release/contributing.md)
+* Reach us on our [ Forums ](https://forums.percona.com/c/mysql-mariadb)
+
+This document describes the workflow for submitting pull requests.
+By contributing, you agree to the [Percona Community code of conduct](https://percona.community/contribute/coc/). Thank
+you for helping us enhance Percona Server for MySQL.
+
+## Why Your Contribution Matters?
+
+Main driving force of open-source project is community. We, as developers team, highly appreciate any contributions.
+
+It may be advisable to initially submit bug fixes to the upstream project that will eventually become part of the next
+release of Percona. Alternatively, Percona provides expedited response times, transparent development processes, and
+streamlined contribution acceptance within a public repository.
+
+## Creating a Jira Issue
+
+The creation of a Jira issue is _mandatory_ first step for a Pool Request preparation. If you're going to write code for some fix or a new feature, you should first describe in form of a Jira ticket what you are going to do. 
+
+A comprehensive description of the bug or feature is essential. It helps the team to perform thorough testing of the solution, to include of pertinent information in the release notes, and for other relevant details. 
+
+Additionally, linking your future pull request to the corresponding issue with a detailed description will facilitate the development process and help reviewers to understand your solution. 
+
+Also, the Jira issue number is a required part of a branch name for your new code.
+
+To create Jira issue, please open the  [ LINK ](https://perconadev.atlassian.net/jira/software/c/projects/PS/issues) and
+authenticate with your favorite social network account (Google, Facebook, etc.). Then follow the instructions and verify
+your e-mail. If Jira starts to redirect you back and forth (their bug, not ours), just open a new browser window with
+the link above.
+
+Now you're ready to an create issue. Find the big "Create" button in the top menu and press it. Choose issue type from
+the second drop-down, saying it is a Bug, an Improvement, or a New feature. Fill in all the required fields below. Please, add accurate and detailed description.
+Good description is a big first step to the solution. The
+rest of the fields are mostly for internal use; the core team developers will fill it later.
+  
+  As soon as Jira issue is submitted, you are ready to start coding.
+  
+## Working with the Source Code
```

Comment:
Same thing - list the steps to work with the source code before I start.

Reply:
Done